### PR TITLE
Add missing contents to  Wave Generator Tutorial

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [ ] Feature implementation
- [x] Doc updates

* **Related Issue**
#72 

* **What changes have you introduced?**
- Add Introduction to the wave generator document
- Add Screenshots of wave generator instrument in the PSLab app.
- Link related experiment video.

* **Does this PR introduce a breaking change?**
No

* **Preview / Steps to verify your work**:
https://github.com/kumuditha-udayanga/pslab-documentation/blob/master/tutorials/wavegenerator.md

### CODE REVIEW

**Comment by kumuditha-udayanga** [comment]
@mariobehling Please review :)

File: tutorials/wavegenerator.md [hunk 1]
```diff
@@ -22,8 +35,19 @@ How to use the Wave Generator
...
+ </table>
+
 Experiment: Generate Sounds with Waves
 --------------------------------------
 
+- Experiment video on Youtube:
+[Using Wave Generator In PSLab Android App to produce different waveforms | FOSSASIA | GSoC'18
+](https://www.youtube.com/watch?v=NC2T5kElWbE)
+
 Experiment: Blinking LEDs with the wave generator
```
Comment (NishithP2004) [inline tutorials/wavegenerator.md]: Why is this Experiment blank?

Comment (Sid911) [reply tutorials/wavegenerator.md]: what? I think that's a link to video for the experiment 😕

Comment (kumuditha-udayanga) [reply tutorials/wavegenerator.md]: @NishithP2004 It was empty so, I added a video that I saw on youtube who does an experiment using the wave generator of PSLab.😇

**Comment by kumuditha-udayanga** [comment]
> Others LGTM

Thank you @NishithP2004 🙂

File: tutorials/wavegenerator.md (hunk 1 above)
Comment (NishithP2004) [reply tutorials/wavegenerator.md]: Actually, my imply was by the Blinking LED experiment.

Comment (NishithP2004) [reply tutorials/wavegenerator.md]: If it's possible please try to fill that as well.

File: tutorials/wavegenerator.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ Wave Generator
...
+In general, waves transfer energy from one location to another, in which case they have a velocity.
+
+![wave-amplitude-length](../_static/wave-amplitude-length.png)
+
+Standing waves may also occur; these have no net velocity and involve no net transfer of energy. They have frequency,
+wavelength and amplitude.
+Mechanical waves require a medium to travel through: for example, sound waves and earthquake waves cannot travel 
+through a vacuum.
+Electromagnetic waves, such as light, do not require a medium and can travel through a vacuum.
+Transverse waves, such as light, oscillate perpendicular to the direction the wave is carrying energy in - as in the
```
Comment (fcartegnie) [inline tutorials/wavegenerator.md]: in - as in ?

Comment (kumuditha-udayanga) [reply tutorials/wavegenerator.md]: Changes a have been done @fcartegnie  🙂

File: tutorials/wavegenerator.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ wavelength and amplitude.
 Mechanical waves require a medium to travel through: for example, sound waves and earthquake waves cannot travel 
 through a vacuum.
 Electromagnetic waves, such as light, do not require a medium and can travel through a vacuum.
-Transverse waves, such as light, oscillate perpendicular to the direction the wave is carrying energy in - as in the
+Transverse waves, such as light, oscillate perpendicular to the direction the wave is carrying energy as in the
```
Comment (fcartegnie) [inline tutorials/wavegenerator.md]: Don't make another commit to fix your commit

Comment (kumuditha-udayanga) [reply tutorials/wavegenerator.md]: @fcartegnie Do you mean to squash the commit?

Comment (kumuditha-udayanga) [reply tutorials/wavegenerator.md]: @fcartegnie Commit is been squashed 🙂
